examples/jsm/nodes/accessors/AutoInstanceNode.js

```diff
@@ -0,0 +1,153 @@
+import Node, { addNodeClass } from '../core/Node.js';
+import { nodeProxy, mat3, mat4 } from '../shadernode/ShaderNode.js';
+import { cameraProjectionMatrix } from './CameraNode.js';
+import { modelViewMatrix, transformedViewMatrix, transformedNormalMatrix, modelNormalMatrix } from './ModelNode.js';
+import { positionLocal } from './PositionNode.js';
+import { normalLocal } from './NormalNode.js';
+import InstanceNode from './InstanceNode.js';
+import { NodeUpdateType } from '../core/constants.js';
+import { instancedBufferAttribute, instancedDynamicBufferAttribute } from './BufferAttributeNode.js';
+import { DynamicDrawUsage, InstancedBufferAttribute, InstancedInterleavedBuffer } from 'three';
+import { instanceIndex } from '../core/IndexNode.js';
+
+class AutoInstanceNode extends Node {
```

Comment:
Would this class extend InstanceNode?

Reply:
I tried it initially, maybe both could come from a base class.